.travis.yml

```diff
@@ -9,25 +9,25 @@ env:
     - XI_BSP_TLS=wolfssl
     - XI_MAKETARGET=tests
   matrix:
-    - PRESET=ARM_REL
-    - PRESET=ARM_REL_MIN
+      # - PRESET=ARM_REL
+      # - PRESET=ARM_REL_MIN
 
-    - PRESET=CC3200
-      XI_MAKETARGET=""
+      #- PRESET=CC3200
+      #XI_MAKETARGET=""
     - PRESET=CC3200_TLS_SOCKET
       XI_MAKETARGET=""
 
-    - PRESET=POSIX_DEV
-    - PRESET=POSIX_DEV_MIN
-    - PRESET=POSIX_REL
-    - PRESET=POSIX_REL_MIN
+      #- PRESET=POSIX_DEV
+      #- PRESET=POSIX_DEV_MIN
+      #- PRESET=POSIX_REL
+      #- PRESET=POSIX_REL_MIN
```

Comment:
Yeah this branch is not yet ready to be reviewed - it is missing some executables from the CC3200 SDK it's still under the testing phase. Sorry for confusion.